multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -220,42 +204,17 @@ write_files:
       # https://github.com/coreos/rkt/issues/2878
       exec nsenter -m -u -i -n -p -t 1 -- /usr/bin/rkt "$@"
 
-{{ if .UseCalico }}
-  - path: /opt/bin/install-calico-system
-    permissions: 0700
-    owner: root:root
-    content: |
-      #!/bin/bash -e
-      /usr/bin/curl -H "Content-Type: application/json" -XPOST -d @"/srv/kubernetes/manifests/calico-system.json" "http://127.0.0.1:8080/api/v1/namespaces"
-
-      /usr/bin/cp /srv/kubernetes/manifests/calico-policy-controller.yaml /etc/kubernetes/manifests
```

Comment:
policy-controller is a static pod - I don't think this can be removed (it's not created like an addon)